give/src/components/Carousel/Carousel.js

```diff
@@ -71,12 +86,12 @@ export const CarouselExamples = () => {
         navButtonClasses="cru-button-dot cru-button-solid cru-button-yellow-gray"
         content={[
           {
-            slide: <Image img={{ src: epcot, alt: "EPCOT" }} />,
+            slide: <Image src={lava1} alt="Lava 1" />,
             label: "Slide 1",
             active: true,
           },
           {
-            slide: <Image img={{ src: epcot, alt: "EPCOT" }} />,
+            slide: <Image src={lava2} alt="Lava 2" />,
```

Comment:
Lava instead of Epcot?! Guess that makes sense 🗻

Reply:
Lol! I'm switching out the demo images to match what's in the AEM component documentation.